prow/crier/reporters/github/reporter.go

```diff
@@ -162,8 +165,61 @@ func (c *Client) Report(ctx context.Context, log *logrus.Entry, pj *v1.ProwJob)
 		defer lock.Release(1)
 	}
 
+	pjsToReport := func() []v1.ProwJob {
+		if len(pj.Spec.Refs.Pulls) != 1 {
+			return nil
+		}
+		// find all prowjobs from this PR
+		selector := map[string]string{}
+		for _, l := range []string{kube.OrgLabel, kube.RepoLabel, kube.PullLabel} {
```

Comment:
We can already filter for presubmit jobs here

Reply:
isn't `kube.PullLabel` sufficient for this check?

Reply:
Oh you are right